sigllm/primitives/prompting/huggingface.py

```diff
@@ -47,6 +47,8 @@ class HF:
         padding (int):
             Additional padding token to forecast to reduce short horizon predictions.
             Default to `0`.
+        restrict_tokens (bool):
+            Whether to restrict tokens or not. Default to `True`.
```

Comment:
seems like the default is `False`. In the `__init__`.

I don't mind either, but if it is `False` make sure to change the existing pipeline `mistral_prompter` and `mistral_detector` to change this parameter to true in this JSON configuration.

Reply:
I saw that you did in `mistral_prompter`, just `mistral_detector` left to also change.

Reply:
I still don't see a change to `mistral_detector`